packages/gatsby-mdx/gatsby/on-create-page.js

```diff
@@ -31,4 +40,53 @@ module.exports = async ({ page, actions }, pluginOptions) => {
       )
     );
   }
+
+  /**
+   * Wrap pages that query mdx in a wrapper for pulling in the MDX scope
+   */
+  if (
```

Comment:
Listen for pages that query for MDX through GraphQL and wrap them through the `pageWithMDX` page.